src/Main.cs

```diff
@@ -336,12 +336,6 @@ private static void ReadSettingsFromArgs(string[] args)
 
                         break;
 
-                    case "lastcharactername":
-                    case "lastcharname":
-                        Settings.GlobalSettings.LastCharacterName = value;
```

Comment:
this logic must exist.
ClassicUOLauncher uses it per profile, so it must be handled in some way

Reply:
But why? This isn't a profile setting -- it's an account/server setting. It has nothing to do with specific profile.

It should be checked when ClassicUO knows what the account name, and server is selected.